### ISSUE TITLE
Doesn't strip `version` attribute with value "1.0"

### ISSUE THREAD

**Issue by tigt** [svg/svgo#649]
This is mostly notable because SVGs created with [potrace](https://en.wikipedia.org/wiki/Potrace) have `version="1.0"`.

Instead, svgo appears to round the attribute to `version="1"`.

SVG’s `version` attribute has never been used by browsers, so it should be safe to omit any value it contains.

**Comment by pmb0** [comment]
I use svgo to preproces a SVG for a google AMP project. Stripping `version` to `1` causes `ampvalidator` to throw following error:

```sh
The attribute 'version' in tag 'svg' is set to the invalid value '1'. (see https://www.ampproject.org/docs/reference/spec.html#svg)
```

`1.0` should not be shortened to `1`.

**Comment by FranckFreiburger** [comment]
[hint](https://github.com/webhintio/hint) reports an error about this: `Bad value “1” for attribute “version” on element “svg”.`

**Comment by tigt** [comment]
Should be fixed by #1028
